CesiumGltf/include/CesiumGltf/PropertyTableView.h

```diff
@@ -1220,11 +1306,28 @@ class PropertyTableView {
                 ErrorBufferViewSizeDoesNotMatchPropertyTableCount);
       }
 
-      return PropertyTablePropertyView<PropertyArrayView<T>, Normalized>(
-          propertyTableProperty,
-          classProperty,
-          _pPropertyTable->count,
-          values);
+      if constexpr (!IsMetadataBoolean<T>::value && !Normalized) {
```

Comment:
We shouldn't be calling `getNumericArrayPropertyValues` for `T = bool` so I don't think this check is necessary